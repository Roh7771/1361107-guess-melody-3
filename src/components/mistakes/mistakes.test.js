import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes";

describe(`GameScreen render correctly`, () => {
  it(`with count 0`, () => {
    const tree = renderer.create(
        <Mistakes
          count={0}
        >
        </Mistakes>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`with count 1`, () => {
    const tree = renderer.create(
        <Mistakes
          count={1}
        >
        </Mistakes>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
