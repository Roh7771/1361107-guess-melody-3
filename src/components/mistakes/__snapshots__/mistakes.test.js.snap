// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameScreen render correctly with count 0 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`GameScreen render correctly with count 1 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;
